impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -458,6 +458,170 @@ define(["./_focus"], function(focusUtil) {
 
   }
 
+  function makeAccessibleToolbar(elem) {
+    elem.setAttribute("role", "toolbar");
+    elem.setAttribute("tabindex", "0");
+
+    var firstItem = null;
+    var lastItem = null;
+
+    var toolbarItems = [];
+
+    elem.addEventListener("keydown", onKeyPress);
+    elem.addEventListener("focus", onContainerFocus);
+
+    items = elem.querySelectorAll('.toolbarButton');
```

Comment:
CSS classes preferably use snake case (using `-` to separate words). Is that name taken already for other purposes?